sematic/ui/src/components/RunPanel.tsx

```diff
@@ -107,10 +107,8 @@ export default function RunPanel(props: {
             <Box sx={{ paddingBottom: 3, gridColumn: 1 }}>
               <Box marginBottom={3}>
                 <Typography variant="h6">{selectedRun.name}</Typography>
-                <CalculatorPath
-                  calculatorPath={"ID:" + selectedRun.id.substring(0, 6)}
-                />{" "}
-                &middot;{" "}
+                <CalculatorPath calculatorPath={"ID: " + selectedRun.id} />
```

Comment:
thank God; this has been such a pain to get in the past 😆 . We should ideally make it easy to get the full id from a note as well.

Reply:
Gotta add a Copy button next to it.